docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3'
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: "Dockerfile.api"
+    command: python3 ohp.py --start-api-server
+    environment:
+      - FLASK_ENV=development
+      - DOCKER_ENV=true
+    ports:
+      -  5000:5000
+    volumes:
+        - .:/OWASP-Honeypot
+    links:
+        - mongodb
+    depends_on:
+        - mongodb
+
+  mongodb:
+    image: mongo:latest
+    container_name: "mongodb"
+    environment:
+        - MONGO_DATA_DIR=/data/db
+        - MONGO_LOG_DIR=/dev/null
+    volumes:
```

Comment:
if we save the data on a volume, we can just start mongodb with `docker-compose up`, no need to do it without mongodb.

Reply:
mongodb is starting with `docker-compose up`.

I thought we might need an option to start the API with mongodb running on the host instead. That is why docker-compose-host.yml is there.